services/blockchain-indexer/shared/indexer/blockchainIndex.js

```diff
@@ -510,6 +510,15 @@ const setIndexVerifiedHeight = ({ height }) => keyValueTable.set(INDEX_VERIFIED_
 
 const getIndexVerifiedHeight = () => keyValueTable.get(INDEX_VERIFIED_HEIGHT);
 
+const updateNumJobsInProgress = async () => {
+	const { queue: bullQueue } = indexBlocksQueue;
+	const numJobsInProgress = await bullQueue.getActiveCount() || await bullQueue.getWaitingCount();
+
+	Signals.get('numJobsInProgressUpdate').dispatch(numJobsInProgress);
+};
+
+setInterval(updateNumJobsInProgress, 10 * 1000); // ms
```

Comment:
Can we tie this to chainNewBlock since we are getting a new block on roughly 10 seconds? If not, maybe we can define the job under jobs folder for better consistency?

Reply:
The reason behind not chaining it to `chainNewBlock` block is that we do not want to stress the DB with unnecessary hits when the chain re-organises. However, we can add it as a job though.